Resources/Prototypes/_CD/Entities/Mobs/Species/avali.yml

```diff
@@ -0,0 +1,82 @@
+- type: entity
+  save: false
+  name: Urist McRaptor
+  parent: BaseMobSpeciesOrganic
+  id: BaseMobAvali
+  abstract: true
+  components:
+  - type: HumanoidAppearance
+    species: Avali
+  - type: Hunger
+  - type: Thirst
+  - type: Icon
+    sprite: _CD/Mobs/Species/Avali/parts.rsi
+    state: full
+  - type: Body
+    prototype: Avali
+    requiredLegs: 2
+  - type: Butcherable
+    butcheringType: Spike
+    spawned:
+    - id: FoodMeatChicken
+      amount: 3
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeCircle
+          radius: 0.35
+        density: 95 #tiny
+        restitution: 0.0
+        mask:
+        - MobMask
+        layer:
+        - MobLayer
+  - type: FloorOcclusion
+  - type: Vocal
+    sounds:
+      Male: MaleAvali
+      Female: FemaleAvali
+      Unsexed: MaleAvali
+  - type: Damageable
+    damageContainer: Biological
+    damageModifierSet: Moth # PLACEHOLDER for now :3
+  - type: DamageVisuals
+    damageOverlayGroups:
+      Brute:
+        sprite: Mobs/Effects/brute_damage.rsi
+        color: "#7a8bf2"
+  - type: Bloodstream
+    bloodReagent: AmmoniaBlood
+  - type: MeleeWeapon
+    soundHit:
+      path: /Audio/Weapons/pierce.ogg
+    angle: 30
+    animation: WeaponArcClaw
+    damage:
+      types:
+        Slash: 5
+  - type: Temperature
+    heatDamageThreshold: 310
+    coldDamageThreshold: 223
+    currentTemperature: 278.15
+    specificHeat: 48
+  - type: ThermalRegulator
+    metabolismHeat: 400 # icebirb go brrr
+    radiatedHeat: 100
+    implicitHeatRegulation: 500
+    sweatHeatRegulation: 2000
+    shiveringHeatRegulation: 2000
+    normalBodyTemperature: 278.15
+    thermalRegulationTemperatureThreshold: 10
```

Comment:
I'll look at this more later but they shouldn't be too drastic. If you could explain exactly what these changes do and why I'd appreciate it.

Reply:
damagethreshold is when they start to take damage from the temp, currentTemperature is their starting / spawn temp, specific heat im not too sure

MetabolismHeat is how much their body makes, radiated is how much they give off, implicit is the natural temp regulation, sweat and shiver is the threshold temp changing, normal body temp is what it tries to default to and thermalregulation is how far either way it needs to be before it starts to regulate itself

Reply:
as for why, Avali have ammonia blood, however if we actually set it to that then they would boil if they went on station since the blood would evaporate, so instead its an "ammonia solution" which is possible in lore, either way their body temp is set to ~5C and they overheat at 36C, but they *freeze* at -50. 

They can handle the cold better then the heat, and have a lower body temp